src/plugins/CMakeLists.txt

```diff
@@ -81,6 +81,8 @@ set(MODULE_SRC
 set(MODULE_LINK
     uicomponents
     engraving
+    notation
+    project
```

Comment:
I wonder if these are really necessary. When only using "interfaces", like `INotationWritersRegister`, `INotationWriterPtr`, `INotation`, linking is usually not necessary because all methods of these interfaces are pure virtual methods, so #including already gives all available information.

Reply:
You are right, these are indeed not necessary. I have added this linking because the plugins module now depends on those other modules, but I didn't notice that this dependency does not really require linking. I have removed it now.